com.unity.multiplayer.mlapi/Editor/NetcodeProfilerModule.cs

```diff
@@ -10,9 +10,9 @@ namespace Unity.Netcode.Editor
     internal static class NetcodeProfilerModule
     {
 #if UNITY_2020_2_OR_NEWER && ENABLE_PROFILER
-        private const string k_RpcModuleName = "MLAPI RPCs";
-        private const string k_OperationModuleName = "MLAPI Operations";
-        private const string k_MessageModuleName = "MLAPI Messages";
+        private const string k_RpcModuleName = "Netcode RPCs";
+        private const string k_OperationModuleName = "Netcode Operations";
+        private const string k_MessageModuleName = "Netcode Messages";
```

Comment:
did we have any dependencies on these hardcoded strings? @becksebenius-unity @Rosme 👀

Reply:
nope, should be fine. upgrading users might have funky profiler modules until they restart the editor. probably not an issue